SimCalorimetry/EcalTrigPrimProducers/plugins/EcalTrigPrimAnalyzer.cc

```diff
@@ -120,29 +99,26 @@ void EcalTrigPrimAnalyzer::analyze(const edm::Event &iEvent, const edm::EventSet
     return;
 
   // comparison with RecHits
-  edm::Handle<EcalRecHitCollection> rechit_EB_col;
-  iEvent.getByLabel(rechits_labelEB_, rechit_EB_col);
-
-  edm::Handle<EcalRecHitCollection> rechit_EE_col;
-  iEvent.getByLabel(rechits_labelEE_, rechit_EE_col);
+  const edm::Handle<EcalRecHitCollection> &rechit_EB_col = iEvent.getHandle(ebToken_);
+  const edm::Handle<EcalRecHitCollection> &rechit_EE_col = iEvent.getHandle(eeToken_);
```

Comment:
The Handle is not really needed here as far as I can see. `Event::get` should work as well and is recommended.

Reply:
done